scripts/functions.sh

```diff
@@ -296,7 +296,32 @@ function follow() {
 function copyOutput() {
   echo "Copying collector output files..."
   mkdir -p ./output
+  # output dir may already include files from previous runs
+  oldFlowFile=$(find ./output -name "*txt" | sort | tail -1)
   ${K8S_CLI_BIN} cp -n "$namespace" collector:output ./output
+  flowFile=$(find ./output -name "*txt" | sort | tail -1)
+  if [[ -n "$flowFile" && "$oldFlowFile" != "$flowFile" ]] ; then
```

Comment:
Is there any interest keeping the txt file ? 

Else we can delete it after the json conversion and assume the latest txt file is the new one 
or even apply the json format to all the txt files found under the output/flow folder :smile_cat:

Reply:
> or even apply the json format to all the txt files found under the output/flow folder 😸

I think that would be bit overkill. 

> We can assume the latest txt file is the new one since we delete it after the conversion

I can delete L300 and remove condition to compare.